app/assets/scripts/components/documents/single-view/document-body.js

```diff
@@ -306,9 +306,16 @@ export const atbdContentSections = [
   {
     label: 'Abstract',
     id: 'abstract',
-    render: ({ element, document }) => (
+    render: ({ element, document, referencesUseIndex, atbd }) => (
```

Comment:
This also needs the `editorSubsections` property. This is used to get the subsections present in the field so they can be shown on the side navigation.

```suggestion
    editorSubsections: (document, { id }) =>
      subsectionsFromSlateDocument(document.abstract, id),
    render: ({ element, document, referencesUseIndex, atbd }) => (
```